### PR TITLE
Mekdims "Experience at Ona" Post

### PR DESCRIPTION
**Pull request by COpenda**
To be published after the [RapidPro PR](https://github.com/onaio/blog/pull/105) has been merged.

### CODE REVIEW

**Comment by COpenda** [comment]
@mberg did you see this? I took the interview approach with Mek's post

**Comment by COpenda** [comment]
@rowo please merge if ok.

**Comment by rowo** [comment]
@mberg do you have some screens about Sofia and how she works, and what value she has? That might be good for this post instead of just a picture of Mek.

**Comment by rowo** [comment]
@COpenda Can you make sure to review for capitalization. Regardless of how a brand's logotype is capitalized, most of the time in print it's title case. Also, I think we can lose the first question – the first day one. You can incorporate some elements into the "what was your experience like" section.

**Comment by rowo** [comment]
>had to catch in on some of the bot action
Do you catch in? Or catch on?

**Comment by rowo** [comment]
@COpenda I added an image (where I Photoshop'd you in) and changed the date to Monday. But, I didn't address the issues I talked about above. Let me know when you are through with them.

**Comment by COpenda** [comment]
@rowo those are some serious photoshop skills. I've merged a summary of question one to the experience question.

**Comment by rowo** [comment]
@COpenda can you review for spelling/grammar? Briefly, I saw some capitalization issues.

File: _posts/2016-08-22-meks-internship.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+type: post
+published: true
+---
+
+![Mek Summer at Ona](/assets/images/2016-08-22/mek-post.jpg)<br>
+<i>Mek sits at the center of attention on his last day.</i>
+
+***What projects were you involved in at Ona?***
+
+The core project I was involved in was one on tracking early child development. I assisted in the design of an enumerator bot that could make it easier for health workers to track the development of children under 5 years. With most companies taking the route towards Artificial Intelligence, it comes to no surprise that Ona had to catch on some of the bot action. The bot called ‘Sofia’ interacts with mothers/caregivers via Facebook Messenger or Telegram to collect data about their children. She does this by asking the mother several questions about her child’s health like, the child’s ability to walk on their own or their ability to say certain words. The project is currently being piloted in Indonesia.
```
Comment (pld) [inline _posts/2016-08-22-meks-internship.md]: * "to catch on" is really awkward
* "it comes to" is wrong, should be "it comes as"
* comma separate the phrase "called 'Sofia"
* the comma after "like" should be before it

Comment (COpenda) [inline _posts/2016-08-22-meks-internship.md]: Replacing catch on with "following suit" does that sound better? @pld

**Comment by rowo** [comment]
@COpenda Once all of Peter's points are addressed, please change the date for posting and let me know.

**Pull request merged**